app/assets/javascripts/views/story_view.js

```diff
@@ -579,6 +476,174 @@ module.exports = FormView.extend({
     }
   },
 
+  renderStory: function() {
+
+    this.$el.append(
+      this.makeFormControl(function(div) {
+        $(div).addClass('form-inline');
+        $(div).append(this.makeFormControl({
+          name: 'estimate',
+          label: true,
+          control: this.select("estimate", this.model.point_values(), {
+            blank: I18n.t('story.no_estimate'),
+            attrs: {
+              class: ['story_estimate'],
+              disabled: this.model.notEstimable() || this.isReadonly()
+            }
+          })
+        }));
+        this.makeStoryTypeSelect(div);
+        var story_state_options = [];
```

Comment:
What do you think about moving lines from 496 to 500 to a function? I believe it would improve legibility.